pom.xml

```diff
@@ -26,7 +26,7 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <odooVersion>17.0</odooVersion>
+        <odooVersion>17-1.0.0</odooVersion>
```

Comment:
I think this should remain as it is (17.0)

Reply:
Final the version of the addon will be `${odooVersion}.${projectVersion}` (e.g 17.0.2.3.0)